gtk4/sys/src/lib.rs

```diff
@@ -13228,6 +13289,12 @@ extern "C" {
         show_peek_icon: gboolean,
     );
 
+    //=========================================================================
+    // GtkPasswordEntryBuffer
```

Comment:
Need to be added to the to be generated list in gtk4/Gir.toml

Reply:
ack